### PR TITLE
Adding stock and flow AMRs to data/models

### LINKED ISSUE #35: Add working stock and flow AMRs
Adding a set of SIR-type stock-and-flow model examples that are known to work properly (i.e., can be sampled and simulated) and include observables, and uncertainty in parameter values and initial conditions. This set of models should serve as the gold-standard set of stock-and-flow models for testing.

### PR DESCRIPTION
**Pull request by sabinala**
This PR will add the gold-star stock-and-flow example AMRs to `data/models`.

Closes #35

### CODE REVIEW

File: data/models/SEIRD_stockflow.json [hunk 1]
```diff
@@ -0,0 +1,312 @@
+{
```
Comment (fivegrant) [inline data/models/SEIRD_stockflow.json:1]:
```suggestion
{
  "id": "seird-stockflow",
```

File: data/models/SEIRHDS_stockflow.json [hunk 2]
```diff
@@ -0,0 +1,486 @@
+{
```
Comment (fivegrant) [inline data/models/SEIRHDS_stockflow.json:1]:
```suggestion
{
  "id": "seirhds-stockflow",
```

File: data/models/SEIRHD_stockflow.json [hunk 3]
```diff
@@ -0,0 +1,437 @@
+{
```
Comment (fivegrant) [inline data/models/SEIRHD_stockflow.json:1]:
```suggestion
{
  "id": "seirhd-stockflow",
```

File: data/models/SEIR_stockflow.json [hunk 4]
```diff
@@ -0,0 +1,255 @@
+{
```
Comment (fivegrant) [inline data/models/SEIR_stockflow.json:1]:
```suggestion
{
  "id": "seir-stockflow",
```

File: data/models/SIR_stockflow.json [hunk 5]
```diff
@@ -0,0 +1,181 @@
+{
```
Comment (fivegrant) [inline data/models/SIR_stockflow.json:1]:
```suggestion
{
  "id": "sir-stockflow",
```

**Review by fivegrant — CHANGES_REQUESTED** [review]
This looks good. The only issue is that this breaks the seeding step since these models don't have unique identifiers. Once that's fixed, it should work.

**Comment by sabinala** [comment]
Alright, I've added the unique model ids @fivegrant suggested. Should be good to go!

**Review by fivegrant — APPROVED** [review]
thanks!

**Pull request merged**